lib/src/base.dart

```diff
@@ -156,7 +156,7 @@ class WebViewStateChanged {
 
   WebViewStateChanged(this.type, this.url, this.navigationType);
 
-  factory WebViewStateChanged.fromMap(Map<String, dynamic> map) {
+  factory WebViewStateChanged.fromMap(Map<dynamic, dynamic> map) {
```

Comment:
I would no change the type here, to keep it type safe, I would cast le value passed to `WebViewStateChanged.fromMap` instead